sympy/solvers/solveset.py

```diff
@@ -1616,7 +1616,7 @@ def solveset(f, symbol=None, domain=S.Complexes):
     {2*n*I*pi | n in Integers}
 
     * If you want to use `solveset` to solve the equation in the
-      real domain, provide a real domain. (Using `solveset\_real`
+      real domain, provide a real domain. (Using `solveset`real`
```

Comment:
This should be an underscore, not a backtick. Also, there should be two backticks as this is code.